cmd/utils/cmd.go

```diff
@@ -344,3 +346,141 @@ func ExportPreimages(db ethdb.Database, fn string) error {
 	log.Info("Exported preimages", "file", fn)
 	return nil
 }
+
+// ImportSnapshot imports a batch of snapshot data into the database
+func ImportChainData(db ethdb.Database, fn string, kind string, checker func(key []byte) bool, interrupt chan struct{}) error {
```

Comment:
Maybe `ImportLdbData`? At least if it can be made to import anything into leveldb, it's not purely chaindata. And also, chaindata in general is spread into ancients too, so calling this importchaindata, it sounds like it can also update the ancient data.